### PR TITLE
automate release process

### PR DESCRIPTION
**Pull request by davidchambers**
As proposed in #3

This pull request is similar to ramda/ramda-fantasy#42. It's nice to use the same set of tools in each of the various Ramda projects. :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,12 @@
 {
   "name": "ramda-lens",
-  "version": "1.0.0",
+  "version": "0.0.0",
```
Comment (davidchambers) [inline package.json:3]: xyz updates `"version"` as part of the release commit, so `"0.0.0"` is the most appropriate value for an unpublished project. If we run `npm run release-minor` we'll release v0.1.0, or if we decide to run `npm run release-major` we'll go straight to v1.0.0.

**Comment by davidchambers** [comment]
It looks as though someone enabled Travis for this repository but did not add **.travis.yml**:

> ```
> Could not find .travis.yml, using standard configuration.
> ```

**Comment by scott-christopher** [comment]
> It looks as though someone enabled Travis for this repository but did not add .travis.yml

It's in #2. I'll raise another PR to separate it out on its own.

**Comment by davidchambers** [comment]
> I'll raise another PR to separate it out on its own.

That would be wonderful. Thanks, Scott!

**Comment by scott-christopher** [comment]
See #5

**Comment by scott-christopher** [comment]
:mushroom:

**Comment by buzzdecafe** [comment]
:tada:

now let's publish it!

**Comment by netfishx** [comment]
+1

**Pull request merged**